src/connectors/kokkos/include/impl/Kokkos_Profiling_C_Interface.h

```diff
@@ -0,0 +1,296 @@
+/*
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 3.0
+//       Copyright (2020) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
```

Comment:
@DavidPoliakoff any chance this header can just be contributed under MIT? Otherwise we need to update the `README.md`, etc. to say that there are multiple licenses in this project.  This is BSD 3-clause so it is not a huge deal to include it, just would be easier if you contributed under the usual inbound license.

Reply:
I'll ask on my side, they don't put me in charge of legal things, on account of wanting the lab to continue existing :)

Reply:
@crtrott : thoughts?

Reply:
Okay, I have the go-ahead to do this. @tgamblin , I'm still as organizationally talented as ever, let me know if I've done that incorrectly

Reply:
I read your comment as "change your license boilerplate to ours," but left our copyright and contract garbage in there

Reply:
Let's merge like this and we'll move to SPDX headers like Spack later.  Putting the license at the top of every file is not required so we can make this a lot shorter.